### PR TITLE
Adding links pointing to the new user profile page

### PR DESCRIPTION
**Pull request by mistercrunch**

### CODE REVIEW

File: superset/models.py [hunk 1]
```diff
@@ -149,13 +149,19 @@ def changed_by_fk(cls):  # noqa
             Integer, ForeignKey('ab_user.id'),
             default=cls.get_user_id, onupdate=cls.get_user_id, nullable=True)
 
-    @renders('created_on')
+    def _user_link(self, user):
+        if not user:
+            return ''
+        url = '/superset/profile/{}/'.format(user.username)
+        return '<a href="{}">{}</a>'.format(url, escape(user) or '')
```
Comment (ascott) [inline superset/models.py]: why do we return html here? looks like we should just be returning the url.

Comment (mistercrunch) [inline superset/models.py]: that's for the `list_columns` view of the CRUD generated "Dashboards" and "Slices" page. We're tricking the list view into displaying links.

Comment (ascott) [inline superset/models.py]: gotcha

**Comment by ascott** [comment]
lgtm

File: superset/views.py [hunk 2]
```diff
@@ -1816,13 +1816,21 @@ def fave_dashboards(self, user_id):
...
+            d = {
+                'id': o.Dashboard.id,
+                'dashboard': o.Dashboard.dashboard_link(),
+                'title': o.Dashboard.dashboard_title,
+                'url': o.Dashboard.url,
+                'dttm': o.dttm,
+            }
+            if o.Dashboard.created_by:
+                user = o.Dashboard.created_by
+                d['creator'] = str(user)
```
Comment (bkyryliuk) [inline superset/views.py]: s/ str(user)/'{}'.format(user.username) to be consistent

File: superset/views.py [hunk 3]
```diff
@@ -1905,12 +1913,20 @@ def fave_slices(self, user_id):
...
+        for o in qry.all():
+            d = {
+                'id': o.Slice.id,
+                'title': o.Slice.slice_name,
+                'url': o.Slice.slice_url,
+                'dttm': o.dttm,
+            }
+            if o.Slice.created_by:
+                user = o.Slice.created_by
+                d['creator'] = str(user)
```
Comment (bkyryliuk) [inline superset/views.py]: ditto

**Comment by bkyryliuk** [comment]
LGTM

**Comment by mistercrunch** [comment]
Gonna let that one slip in as is. We can always alter it later.

**Pull request merged**